csrankings-t.csv

```diff
@@ -636,6 +636,7 @@ Tobias Hoßfeld,University of Würzburg,http://www.comnet.informatik.uni-wuerzbu
 Tobias Knopp,Hamburg University of Technology,https://www.tuhh.de/ibi/people/tobias-knopp-head-of-institute.html,5gpkOBAAAAAJ
 Tobias Kramer,University of Koblenz-Landau,https://www.uni-koblenz-landau.de/de/koblenz/fb4/ifm/mud/team/tobias-kraemer,19ZQCsMAAAAJ
 Tobias Kuhn,VU Amsterdam,http://www.tkuhn.org,26_NNqEAAAAJ
+Tobias Langlotz,University of Otago,https://www.otago.ac.nz/info-science/people/staff/tobias-langlotz.html,2KnwGqUAAAAJ

```

Comment:
As above, info sciences, not CS.

Reply:
There are two computing departments here in Otago that work very closely together, sharing teaching and PhD supervision. These are Computer Science and Information Science. Tobias Langlotz is an internationally well-known expert in VR and HCI, both categories in the csranking. So I think he needs to be included otherwise the csranking would not reflect the expertise in Otago.

Reply:
See above. If and only if both faculty can solely supervise PhDs in the Computer Science department, then they are eligible for inclusion.